client/src/partials/finance/reports/agedCreditors/agedCreditors.html

```diff
@@ -0,0 +1,33 @@
+<div class="flex-header static">
```

Comment:
I don't think this file should exist.  Normally, for a report, all you need is two client file:
 1. `report.config.js`
 2. `report.modal.html`

Why does this file exist?